CONTRIBUTING.md

```diff
@@ -78,6 +78,25 @@ If you plan to submit a pull request for anything more than a typo or obvious bu
 Depending on the size of the feature you may be expected to first write a design proposal.
 A proposal template is [available here](https://github.com/heptio/contour/tree/master/design/design-document-tmpl.md)
 
+### Rules of thumb for commits
+
+Generally, we try to have our commits stick to the following rules of thumb:
+
+- Have a short subject on the first line and a body. The body can be empty.
```

Comment:
I'd like to see something that can be copy pasted as a template

Fixes (or Updates): #NNNN


Signed-off-by: Your Name <you@youremail.com>
```
packagename: what will happen when


```

Then bullet points explaining what we meant

- The subject line should explain what will happen if this change is applied. Use the imperative mood (ie "If applied, this commit will (subject)" should make sense). The subject line should _not_ explain _how_ the change does what it does. For instance "fixes typo" is better than "change spelling of misspelled word".
- The body of the commit message should describe _why_ the change is needed and give background if necessary. The body should not spend time describing _how_ the change does what it does, that's already evident from the diff of the change.

Reply:
Added a template and a sample.